app/src/main/java/org/oppia/android/app/profile/AdminPinActivityPresenter.kt

```diff
@@ -38,9 +38,10 @@ class AdminPinActivityPresenter @Inject constructor(
 
   /** Binds ViewModel and sets up text and button listeners. */
   fun handleOnCreate() {
-    activity.supportActionBar?.setDisplayHomeAsUpEnabled(true)
-    activity.supportActionBar?.setHomeAsUpIndicator(R.drawable.ic_close_white_24dp)
-    activity.supportActionBar?.setHomeActionContentDescription(R.string.admin_auth_close)
+
+//    activity.supportActionBar?.setDisplayHomeAsUpEnabled(true)
+//    activity.supportActionBar?.setHomeAsUpIndicator(R.drawable.ic_close_white_24dp)
+//    activity.supportActionBar?.setHomeActionContentDescription(R.string.admin_auth_close)
```

Comment:
It is very important for accessibility to set this ContentDescription value either here or in xml.